src/app/docs/icon/icon-docs.component.html

```diff
@@ -4,143 +4,170 @@
-      heading="Dictionary"
+      heading="Accessibility"
     >
-      <p>
-        The icon dictionary provides a complete list of the Font Awesome icons that we recommend for use in SKY UX. We support <a href="https://fontawesome.com/v4.7/icons/">Font Awesome version 4.7.0</a>. If Font Awesome does not have a particular icon, it is probably not common enough for its purpose to be immediately obvious to users.
+      <p class="sky-docs-text-block">
+        Icons in SKY UX should be used in a way that meet <a href="https://www.w3.org/WAI/WCAG21/Techniques/general/G207.html" target="_blank">WCAG contrast standards</a>. Icons must meet a 3:1 contrast ratio with the background on which they're applied. 
       </p>
     </sky-docs-demo-page-section>
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+    <sky-docs-design-guideline
+    heading=""
+    headingStyle="success"
+  >
+    <p>Icons should have meaningful text labels where possible.</p>
+    <sky-docs-design-guideline-thumbnail
+      caption="Icons should have meaningful text labels where possible."
+      captionType="success"
+      imageSource="~/assets/img/guidelines/icons/icon-usage-1.png"
+    >
+    </sky-docs-design-guideline-thumbnail>
+    <sky-docs-design-guideline-thumbnail
+      caption="Avoid standalone icons when possible."
+      captionType="danger"
+      imageSource="~/assets/img/guidelines/icons/icon-usage-2.png"
+    >
+    </sky-docs-design-guideline-thumbnail>
+  </sky-docs-design-guideline>
+
+  <sky-docs-design-guideline
+  heading=""
+  headingStyle="success"
+>
+  <p>Avoid using icons solely for visual interest or decoration.</p>
+  <sky-docs-design-guideline-thumbnail
+    caption="Don't icons solely for visual interest or decoration."
+    captionType="success"
+    imageSource="~/assets/img/guidelines/icons/icon-usage-3.png"
+  >
+  </sky-docs-design-guideline-thumbnail>
+  <sky-docs-design-guideline-thumbnail
+    caption="Don't icons solely for visual interest or decoration."
+    captionType="danger"
+    imageSource="~/assets/img/guidelines/icons/icon-usage-4.png"
+  >
+  </sky-docs-design-guideline-thumbnail>
+</sky-docs-design-guideline>
+    
+<sky-docs-design-guideline
+heading=""
+headingStyle="success"
+>
+<p>Do not use icons with established meanings for different purposes.</p>
+<sky-docs-design-guideline-thumbnail
+  caption="Only use icons that have established meanings with the label with which they are paired."
+  captionType="success"
+  imageSource="~/assets/img/guidelines/icons/icon-usage-5.png"
+>
+</sky-docs-design-guideline-thumbnail>
+<sky-docs-design-guideline-thumbnail
+  caption="Do not use icons with established meanings for different purposes."
+  captionType="danger"
+  imageSource="~/assets/img/guidelines/icons/icon-usage-6.png"
+>
+</sky-docs-design-guideline-thumbnail>
+</sky-docs-design-guideline>
+  
+    </sky-docs-demo-page-section>
+    <sky-docs-demo-page-section
+    heading="Contributing">
```

Comment:
Seems a little odd to me for the header for the table of icons to be "Contributing." Should this go back to "Dictionary" or whatever we have in place now?

Reply:
The "contributing" section isn't related too inventory below; I'll add a heading to preceed that, since I can see how that might be confusing.